Editor/Data/Attribute/CustomInspectorTextAttributeDrawer.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Zinnia.Data.Attribute
+{
+    using UnityEngine;
+    using UnityEditor;
+
+    [CustomPropertyDrawer(typeof(CustomInspectorTextAttribute))]
+    public class CustomInspectorTextAttributeDrawer : PropertyDrawer
+    {
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            EditorGUI.PropertyField(position, property, new GUIContent((attribute as CustomInspectorTextAttribute).customText));
```

Comment:
```suggestion
            EditorGUI.PropertyField(position, property, new GUIContent(((PropertyDrawer)attribute).customText));
```

We know it's that type, due to the `CustomPropertyDrawer` attribute telling Unity that.

Reply:
![image](https://user-images.githubusercontent.com/36199993/66707834-2a4e2e00-ed3f-11e9-8e64-0eb88706e5c1.png)

this change throws an error

Reply:
Ha damn, that was a copy and past error on my behalf! 😄 I edited the suggestion (because GitHub doesn't seem to allow suggestions in comments down the comment thread chain).